File: Data Structures & Algorithms/evaluate-reverse-polish-notation/submission-0.cpp
class Solution {
public:
    int evalRPN(vector<string>& tokens) {
        int n  = tokens.size();
        int a  = 0;
        int b = 0;
        int c = 0;
        int pushing = 0;
        stack<int> ans;
        for(int i=0;i<n;i++){
           if(tokens[i]=="+" || tokens[i]=="-" || tokens[i]=="*" || tokens[i]=="/"){
            a=ans.top();
            ans.pop();
            b=ans.top();
            ans.pop();
            if(tokens[i]=="+"){
                c = b+a;
            }
            if(tokens[i]=="-"){
                c = b-a;
            }
            if(tokens[i]=="*"){
                c = b*a;
            }
            if(tokens[i]=="/"){
                c = b/a;
            }
            ans.push(c);
           }
           else{
            pushing = std::stoi(tokens[i]);
            ans.push(pushing);
           }
        }
        int final = ans.top();
        return final;
    }
};

File: Data Structures & Algorithms/baseball-game/submission-0.py
class Solution:
    def calPoints(self, operations: List[str]) -> int:
        ans = []
        
        for op in operations:
    
            if op == "+":
                ans.append(ans[-1]+ans[-2])
            elif op == "C":
                ans.pop()
            elif op == "D":
                ans.append(2*ans[-1])
            else:
                ans.append(int(op)) 

        return sum(ans)

        